packages/orchestrator/internal/sandbox/checks.go

```diff
@@ -118,6 +114,13 @@ func (s *Sandbox) LogMetrics(ctx context.Context) {
 	}
 }
 
+func (s *Sandbox) LogSignals(ctx context.Context) {
+	if isGTEVersion(s.Sandbox.EnvdVersion, minEnvdVersionForMetrics) {
```

Comment:
why do we need to check for minEnvdVersionForMetrics, there's no requirement on envd, is it?